Initial Abstract (#58)
MrLongbottom/P9-Rapport: sections/Abstract.tex

> We investigate whether using a topic model can improve \gls{ir} methods.

Reviewer: ```suggestion
Topic models are used to discover topics within a set of documents.
In this paper, we investigate whether using a topic model can improve existing \gls{ir} methods.
```

> We employ a probabilistic topic model, specifically \gls{lda}, to automatically extract topics from Danish news articles.

Reviewer: ```suggestion
We employ the probabilistic topic model, \gls{lda}, to automatically extract topics from Danish news articles.
```

%How can \gls{pr} be used on a dataset without explicit edges?

> We use the topic distribution, from the \gls{lda}, to construct the adjacency matrix edge by measuring the Jensen-Shannon distance between each article topic distribution, since references between news articles do not exist.

Reviewer: ```suggestion
We use the topic distribution, from the \gls{lda}, to construct an adjacency matrix, by measuring the Jensen-Shannon distance between the articles' topic distributions.
```

> We use \gls{lda}, \gls{tf-idf}, \gls{bm25}, \gls{lm} and \gls{pr} as our baselines for testing \gls{ir} performance.

Reviewer 2: ```suggestion
We use \gls{lda}, \gls{tf-idf}, \gls{bm25}, \gls{lm}, and \gls{pr} as our baselines for testing \gls{ir} performance.
```

> We also combine these in various ways to investigate the individual performance of each method.

Reviewer 2: Hmm, viser kombinationerne virkelig modellerne 'individuelle' performance?

We evaluate the \gls{ir} methods by generating document and topic queries which are based on the topic distribution from the \gls{lda}.

> \gls{bm25} is the best performing baseline, we have, and we improve \gls{bm25} by combining it with \gls{pr}.

Reviewer: Jeg tror gerne at jeg først vil snakke om hvordan vi prøver på at evaluere på to måder / en ny måde og først derefter slutte af med algoritmerne.
Men har svært ved at få det formuleret...

Reviewer: ```suggestion
We test various \gls{ir} methods' ability to find documents related to a query, but also to find documents related to topics expressed in queries.
We then evaluate \gls{ir} performance of combinations of \gls{lda}, \gls{tf-idf}, \gls{bm25}, \gls{lm} and \gls{pr}.
```
